pkgs/development/python-modules/colbert-ai/default.nix

```
@@ -0,0 +1,66 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+
+  # build-system
+  setuptools,
+
+  # dependencies
+  bitarray,
+  datasets,
+  flask,
+  python-dotenv,
+  ninja,
+  scipy,
+  tqdm,
+  transformers,
+  ujson,
+  gitpython,
+  torch,
+  faiss,
+}:
+
+buildPythonPackage rec {
+  pname = "colbert-ai";
+  version = "0.2.21";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "colbert_ai";
+    hash = "sha256-qNb9tOInLysI7Tf45QlgchYNhBXR5AWFdRiYt35iW6s=";
+  };
+
+  pythonRemoveDeps = [ "git-python" ];
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    bitarray
+    datasets
+    flask
+    gitpython
+    python-dotenv
+    ninja
+    scipy
+    tqdm
+    transformers
+    ujson
+    torch
+    faiss
+  ];
+
+  pythonImportsCheck = [ "colbert" ];
+
+  meta = {
+    description = "ColBERT is a fast and accurate retrieval model, enabling scalable BERT-based search over large text collections in tens of milliseconds.";
```

```suggestion
    description = "Fast and accurate retrieval model, enabling scalable BERT-based search over large text collections in tens of milliseconds";
```

> - Not start with definite (“The”) or indefinite (“A”/“An”) article.
> - Not start with the package name.
>   - More generally, it should not refer to the package name.
> - Not end with a period (or any punctuation for that matter).

https://nixos.org/manual/nixpkgs/unstable/#var-meta-description

---

Fixed.